### PR TITLE
[docs] Update Android dev environment setup

### PR DESCRIPTION
**Pull request by pastk**
Signed-off-by: Konstantin Pastbin <konstantin.pastbin@gmail.com>

### CODE REVIEW

**Comment by pastk** [comment]
Base on my experience setting up an android dev env on Linux (Ubuntu 18.04).

Not sure all steps will be same for MacOS & Win.

File: docs/INSTALL.md [hunk 1]
```diff
@@ -227,6 +227,17 @@ export PATH=$HOME/Android/Sdk/cmake/3.18.1/bin:$PATH
 export PATH=$HOME/Library/Android/Sdk/cmake/3.18.1/bin:$PATH
 ```
 
+Check if you have a system-wide Java runtime environment (JRE) installed:
+```bash
+java --version
+```
+
+If your system doesn't have a JRE installed or you want command line builds to use a JRE version bundled with the Studio then set `JAVA_HOME` environment variable:
```
Comment (vng) [inline docs/INSTALL.md:235]: For Mac OS:
```bash
java -version
```
 ... or Java version is less than 11 (openjdk) ..
```bash
JAVA_HOME=*A-PATH-TO-YOUR-ANDROID-STUDIO-INSTALLATION*/Contents/jre/Contents/Home
```

**Review by vng — COMMENTED** [review]
Added instructions for MacOS

File: docs/INSTALL.md [hunk 2]
```diff
@@ -190,21 +190,21 @@ Some tests [are known to be broken](https://github.com/organicmaps/organicmaps/i
...
-- Choose "Android 11 (R) API Level 30" SDK.
-- Choose "version "30" and click "OK".
+- Open "SDK Manager" (under "More Actions" in a welcome screen or a three-dot menu in a list of recent projects screen or "Tools" top menu item in an open project).
+- Select "Android 11.0 (R) / API Level 30" SDK.
+- Switch to "SDK Tools" tab
 - Check "Show Package Details" checkbox.
-- Choose "NDK (side by side)" version **21.X.Y**.
-- Choose "CMake" version **3.18.1**.
-- Click "OK".
+- Select "NDK (Side by side)" version **21.X.Y**.
```
Comment (biodranik) [inline docs/INSTALL.md:205]: NDK 21.4.7075529 should be used, as it is defined in build.gradle

Comment (pastk) [reply docs/INSTALL.md:205]: Will fix that, thanks!
As a side-note: is it a normal practice to keep versions hardcoded like that in build.gradle? Is it possible to move it to gradle.properties?

File: docs/INSTALL.md (hunk 1 above)
Comment (pastk) [reply docs/INSTALL.md:235]: Thanks, I'll fix that!
Actually in linux its also possible to use `java -version` (single dash)

File: docs/INSTALL.md (hunk 2 above)
Comment (biodranik) [reply docs/INSTALL.md:205]: This is the latest NDK that works with our code. There is a complex compilation issue with newer NDKs, so at the moment it is the only working practice :)

**Review by biodranik — APPROVED** [review]

**Pull request merged**